src/main/resources/application.properties

```diff
@@ -123,7 +123,7 @@ ldap.server.list: [{\
 spring.mongodb.embedded.version: 3.4.1
 # We remove the embedded mongodb in tests since in most of them we set up our own before Spring
 # starts and activate it manually in tests where we need the Spring's own embedded mongo DB
-spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration
+spring.autoconfigure.exclude: org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration
 
 # Threading settings
 threads.corePoolSize: 100
```

Comment:
change to lower letters of 'P' and 'S'. And add '.' or '-' as separator between words.
Or is this property some library property and can't be changed?
Same comment for the other threads properties.

Reply:
Done

Reply:
ok